examples/js/margin-borrow-buy-sell-repay-example.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+const ccxt = require ('../../ccxt.js');
+
+// AUTO-TRANSPILE //
+
+// Note: examples or implementations are subject to possible change in future. Please subscribe to CCXT-announcements telegram/discord channel to be informed of related & important updates.
+
+async function example () {
+    const exchange = new ccxt['binance']({'apikey': 'xxx', 'secret': 'yyy'});
+    // set target symbol
+    const symbol = 'BUSD/USDT';
+    // which asset you want to use for margin-borrow collateral
+    const collateral_coin = 'USDT'; 
+    // which coin to borrow
+    const borrow_coin = 'BUSD'; 
+    // how many coins to sell
+    const amount_to_trade = 20;
+    // what is the target margin. This might be obtainable from exchange automatically, using available endpoints, but for example purposes, we set here manually
+    const margin_magnitude = 10; 
+    // for example purposes, let's also check available balance at first
+    const balance_margin = await e.fetchBalance ({'defaultType': 'margin', 'marginMode': 'isolated'}); // 'isolated' or 'cross'.
```

Comment:
I think there's a typo here and everywhere below, we should use `exchange` instead of `e`, no?

```suggestion
    const balance_margin = await exchange.fetchBalance ({'defaultType': 'margin', 'marginMode': 'isolated'}); // 'isolated' or 'cross'.
```

Reply:
ogrhhh, have moved snippet back and forth and forgot to change variable name back.  fixed.